crates/wascc-provider/src/states.rs

```diff
@@ -1,6 +1,5 @@
 pub(crate) mod crash_loop_backoff;
 pub(crate) mod error;
-pub(crate) mod finished;
```

Comment:
As a note: These finished states in each provider were never used or transitioned to, so I removed them